newIDE/app/src/ExpressionAutocompletion/ExpressionAutocompletion.spec.js

```diff
@@ -242,17 +242,11 @@ describe('ExpressionAutocompletion', () => {
       expect(autocompletions).toEqual(
         expect.arrayContaining([
           expect.objectContaining({
-            completion: 'Animation',
+            completion: 'AnimationFrameCount',
```

Comment:
When you showed us the demo, I felt like this returned `Animation::Name` and something else like `Animation::...`, but I'm not sure

Reply:
This is animation related, but it's specific to Sprites (models don't have frames).
I plan to add expression for `Duration` and `ElapsedTime` in the capability in near future.

Reply:
Oh ok I see

Reply:
Maybe a TODO to add somewhere?

Reply:
There is a list in the description. I think it's not technical or critical enough to need a todo.